modules/ROOT/pages/update-acm.adoc

```diff
@@ -255,6 +256,41 @@ If you are upgrading from a package version before 1.80.1, update the permission
 .. Save.
 . Verify that your permissions match those in the xref:create-community.adoc#configure-profiles-and-permissions[Configure Profiles and Permissions] documentation.
 
+Return to the main list of upgrade instructions to continue.
+
+[[post-182]]
+== Postinstallation Steps When Upgrading from a Version Before 1.82
+
+If you are upgrading from a package version before 1.82, use either the automatic or manual configuration methods to update the permissions of the *Member* and *Guest* profiles on the *AnypointAssetCustomFields* and *AnypointCustomFields* objects, and ensure both *Member* and *Guest* profiles have field-level *Read* access to the *AssetVersionId* field in the *AnypointApiInstances* object:
+
```

Comment:
Instead of one big task, separate automatic and manual configuration.

Reply:
Fixed.